install.sh

```diff
@@ -20,12 +20,12 @@ source install/check-latest-commit.sh
 source install/check-minimum-requirements.sh
 
 # Let's go! Start impacting things.
-# Upgrading clickhouse needs to come first before turning things off, since we need the old clickhouse image
+source install/create-docker-volumes.sh
+source install/update-docker-volume-permissions.sh
+# Upgrading clickhouse needs to come before turning things off, since we need the old clickhouse image
 # in order to determine whether or not the clickhouse version needs to be upgraded.
 source install/upgrade-clickhouse.sh
```

Comment:
I placed this strategically here, because upgrading clickhouse requires knowledge of past versions of clickhouse before proceeding. The only way I could foresee doing that would be to check if a past clickhouse volume exists, then bring up clickhouse to check the version.

Are you encountering errors during installation? I would expect the `upgrade-clickhouse.sh` script to be skipped completely for you

Reply:
maybe there's stuff from sentry dev that's causing this to be run?

Reply:
> upgrading clickhouse requires knowledge of past versions of clickhouse before proceeding

ahh i see

> Are you encountering errors during installation? 

no, i was just trying to reproduce the linked issue and it only happens if the volume's deleted (somehow), it makes sense now that it's required, so not an issue on our part